mappings/net/minecraft/client/gui/hud/ChatHud.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_5581 net/minecraft/client/gui/hud/ChatHud
-	FIELD field_27194 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_27194 CHAT_HUD_LOGGER Lorg/apache/logging/log4j/Logger;
```

Comment:
CHAT_HUD_LOGGER is redundant and therefore not necessary, see [naming conventions](https://github.com/Legacy-Fabric/yarn/blob/1.8.9/CONVENTIONS.md#conciseness).
Same goes for the other loggers.

Reply:
This causes an inheritance issue between Screen and its children, so naming them specifically helps to aleviate it.

Reply:
no it doesn't?

Reply:
Yes it does. It's the reason the logger in `Screen` was never mapped. Every other logger in its children was deemed more important.

Reply:
wtf the logger in Screen is marked as mapped in my Enigma!? Now I am confused...

Reply:
> This causes an inheritance issue between Screen and its children, so naming them specifically helps to aleviate it.

this looks like an enigma bug

Reply:
since static fields aren't inherited

Reply:
Reverted these changes and renamed only `Screen`'s Logger instead.